### ISSUE TITLE
WIDGET_*: Setting keyword to undefined value handled wrongly

### ISSUE THREAD

**Issue by jtappin** [gnudatalanguage/gdl#1051]
In the widget routines, setting the `SENSITIVE` keyword to an undefined value should (at least for consistency with IDL) be equivalent to not setting it at all, whereas it is treated as being set to zero. The attached routine: 
[qd_base_s.pro](https://github.com/gnudatalanguage/gdl/files/6982108/qd_base_s.txt) illustrates the problem.

````
GDL> qd_base_s            ; All buttons insensitive (should be sensitive)
GDL> qd_base_s, /sensitive           ; All buttons sensitive (correct)
GDL> qd_base_s, sensitive=0         ; All buttons insensitive (correct)
````
I've not done a thorough check to see if the behaviour is common to all keywords and if it extends beyond the widget routines.
Current GIT (== 1.0.0) on Manjaro Linux

**Comment by GillesDuvert** [comment]
It's a general GDL problem with 'logical' keywords, and should not be restricted to this case. There may be several others in the code. Thanks!

**Comment by jtappin** [comment]
I'd conjecture it's only really a problem where a logical keyword has a default of TRUE and is set explicitly to zero to disable the option .

**Comment by GillesDuvert** [comment]
solved in #1054
